msvc/scripts/genconsts.cs

```diff
@@ -0,0 +1,98 @@
+using System;
```

Comment:
I have another idea.
This C# program just runs winsetup.bat and extracts one line, into a file with a "template" aroun that? Can you just augment winsetup.bat to do all this work? It'd only be like 2 or 3 more lines and remove the C# entirely?